convention.md

```diff
@@ -282,11 +282,15 @@ Recommended unit strings:
 
 You are not limited to the recommended values, although they are the only well known ones that will have to be recognized by any Homie consumer.
 
-* `homie` / `device ID` / `node ID` / `property ID` / **`set`**: the device can subscribe to this topic if the property is **settable** from the controller, in case of actuators.
+#### Property command topic
 
-Homie is state-based.
-You don't tell your smartlight to `turn on`, but you tell it to put its `power` state to `on`.
-This especially fits well with MQTT, because of retained message.
+* `homie` / `device ID` / `node ID` / `property ID` / **`set`**: The device must subscribe to this topic if the property is **settable** (in case of actuators for example).
+
+The assigned and processed payload must be reflected by the Homie device in the property topic `homie` / `device ID` / `node ID` / `property ID` as soon as possible.
+
+A Homie controller publishes to the command topic with non-retained messages only.
+
+Homie is state-based. You don't tell your smartlight to `turn on`, but you tell it to put its `power` state to `on`.
```

Comment:
I'd vote to actually remove those two sentences. By now they do not add any real value.